src/pages/PushNofiPage/notifi.tsx

```
@@ -0,0 +1,57 @@
+import Header from '@/components/Header';
+import { FILLBOM_COLOR } from '@/constants/color';
+import NoPatients from './components/NoPatients';
+import PatientCardAccepted from './components/PatientCardAccepted';
+import PatientCardPending from './components/PatientCardPending';
+import useGetPatientList from './hooks/useGetPatientList';
+import { patientCardStyles, styles } from './styles';
+
+const PatientListPage = () => {
+  const result = useGetPatientList();
+  const data = result?.data;
+  const isLoading = result?.isLoading ?? false;
```

ESLint

[error] 'isLoading' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)